apps/datafeeder/src/app/presentation/components/data-import-validation-map-panel/data-import-validation-map-panel.component.ts

```diff
@@ -10,19 +10,20 @@ import {
   ViewChild,
 } from '@angular/core'
 import { ThemeService } from '@geonetwork-ui/util/shared'
-import type { Feature } from 'geojson'
+import type { Feature as FeatureType } from 'geojson'
 import { asArray, asString } from 'ol/color'
 import { isEmpty } from 'ol/extent'
 import GeoJSON from 'ol/format/GeoJSON'
 import { Geometry } from 'ol/geom'
 import TileLayer from 'ol/layer/Tile'
 import VectorLayer from 'ol/layer/Vector'
-import Map from 'ol/Map'
+import { Map, Feature } from 'ol'
```

Comment:
It's generally safer to import things from more "precise" entrypoints in ol, to avoid bringing too much of the library. E.g.:

```ts
import Map from 'ol'
import Feature from 'ol/Feature'
```